index.js

```diff
@@ -142,16 +141,34 @@ function replaceEnvVars(config) {
     });
 }
 
+function requiresConfig(options) {
+    return !options.generate && (options.start || options.test);
+}
+
 function main() {
     const options = getOptions();
-    let configFile = options.configFile;
-    if (!/^\//.test(configFile)) {
+
+    let configFilePath = options.configFilePath || 'config.yaml';
+    if (!/^\//.test(configFilePath)) {
         // resolve relative paths
-        configFile = path.resolve(`${process.cwd()}/${configFile}`);
+        configFilePath = path.resolve(`${process.cwd()}/${configFilePath}`);
```

Comment:
Hm... The `app_base_path` is the path to the deployment root path. Config shouldn't really be there..